packages/salesforcedx-utils-vscode/package.json

```diff
@@ -9,14 +9,16 @@
   "categories": ["Other"],
   "dependencies": {
     "rxjs": "^5.4.1",
-    "tree-kill": "^1.1.0"
+    "tree-kill": "^1.1.0",
+    "which": "1.3.0"
```

Comment:
This package helps us determine which sfdx is running, the sfdx.exe or the sfdx.cmd one.